src/test/java/spoon/test/template/TemplateTest.java

```diff
@@ -727,64 +728,14 @@ public void substituteStringLiteral() throws Exception {
 	public void substituteSubString() throws Exception {
 		//contract: the substitution of substrings works on named elements and references too
 		Launcher spoon = new Launcher();
-		spoon.addTemplateResource(new FileSystemFile("./src/test/java/spoon/test/template/testclasses/SubStringTemplate.java"));
```

Comment:
I'm not sure why you deleted all those lines in this test: the original contracts remain true, as SubstringTemplate contain a `@Parameter`, no?